src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -911,7 +914,7 @@ private AuthenticationTicket ValidateToken(string idToken, OpenIdConnectMessage
                 protocolValidationContext.AuthorizationCode = message.Code;
```

Comment:
This if clause, is not needed as the ProtocolValidationContext has the entire message. Also, the Property 'AuthorizationCode' is marked for to be removed in beta8.